src/components/app.test.ui.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * @jest-environment jsdom
+ */
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+
+import {Provider} from 'react-redux';
+import {configureStore} from '../store';
+import {App} from './app';
+
+describe('Voyager', () => {
+
+  describe('instantiation via component', () => {
+    test('renders voyager', done => {
```

Comment:
Very minor point, but we mostly use `describe` and `it` elsewhere as it's more consistent with BDD style.  (If I understand correctly, `test` is used more with `suite` -- but yeah this is very minor as they are equivalent.)